tests/LanguageServerTest.php

```diff
@@ -36,7 +36,10 @@ public function testInitialize()
                 'textDocumentSync' => TextDocumentSyncKind::FULL,
                 'documentSymbolProvider' => true,
                 'hoverProvider' => true,
-                'completionProvider' => null,
+                'completionProvider' => (object)[
+                    'resolveProvider' => false,
+                    'triggerCharacters' => ['$', '>']
```

Comment:
Only characters are allowed in the array unfortunately. Using `->` will trigger the completion on `-` instead, as VS Code only looks at the first char. We should open an issue at LSP to support strings, this is a pretty big limitation for PHP and C++. `:` should be added